xcconfigs/Shopist.xcconfig

```diff
@@ -1,4 +1,6 @@
 #include "Datadog.xcconfig"
 
+// TODO: RUMM-664 temporary value, otherwise test cases fail in CI
+RUM_APPLICATION_ID=12345
```

Comment:
Like said, what about also including the `DATADOG_CLIENT_TOKEN=` here 🙂? This will be more obvious on what to fill in when creating the `Sophist.local.xcconfig`:
```suggestion
RUM_APPLICATION_ID=12345 // use your own RUM Application ID obtained on datadoghq.com
DATADOG_CLIENT_TOKEN=// use your own Client Token, generated for RUM_APPLICATION_ID
```

Reply:
yes makes sense 👍 if you don't provide it in `local.xcconfig` you will get fatalError at runtime 👌